src/components/Keys/Keys.jsx

```diff
@@ -0,0 +1,100 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
+import PropTypes from 'prop-types';
+import React from 'react';
+import './Keys.css';
+
+const Keys = ({ handleButtonClick }) => (
+  <div>
+    <div className="row">
+      <button type="button" onClick={() => handleButtonClick('AC')}>
+        AC
+      </button>
+      <button type="button" onClick={() => handleButtonClick('+/-')}>
+        +/-
+      </button>
+      <button type="button" onClick={() => handleButtonClick('%')}>
+        %
+      </button>
+      <button
+        type="button"
+        className="far-right"
+        onClick={() => handleButtonClick('÷')}
+      >
+        ÷
+      </button>
+    </div>
+    <div className="row">
+      <button type="button" onClick={() => handleButtonClick('7')}>
+        7
+      </button>
+      <button type="button" onClick={() => handleButtonClick('8')}>
+        8
+      </button>
+      <button type="button" onClick={() => handleButtonClick('9')}>
+        9
+      </button>
+      <button
+        type="button"
+        className="far-right"
+        onClick={() => handleButtonClick('x')}
+      >
+        X
+      </button>
+    </div>
+    <div className="row">
+      <button type="button">4</button>
+      <button type="button">5</button>
+      <button type="button">6</button>
```

Comment:
- [ ] You must have forgotten to add the onClick event to these buttons, kindly fix this.